src/lib/payment-processing/providers/radom/index.ts

```diff
@@ -0,0 +1,46 @@
+import { env } from '$env/dynamic/private';
+import { error } from '@sveltejs/kit';
+import type { Radom } from './types';
+import type { SessionDetails } from '$lib/payment-processing/types';
+
+export * from './types';
+
+export * from './adapter';
+
+export const PROVIDER_NAME = 'radom';
+
+export async function radomApi<T = any>(resourcePath: string, options?: RequestInit): Promise<T> {
```

Comment:
Can you add a comment here saying this code should only be called by backend code?

I want to avoid us accidentally using this in front end code and leaking the API key